Envfile

```diff
@@ -155,6 +155,9 @@ variable :TWITCH_WEBHOOK_SECRET, :String, default: "Optional"
 # Trending tags on home page
 variable :TRENDING_TAGS, :String, default: "git,beginners"
 
+# For calling the Stack Exchange API
+variable :STACKEXCHANGE_APP_KEY, :String, default: ""
```

Comment:
Maybe `STACK_EXCHANGE_APP_KEY` ? After all the name of the product is Stack Exchange stylized as StackExchange